src/supervisor/worker.js

```diff
@@ -168,27 +170,28 @@ function main () {
     });
 
     // Only start watching for file changes if the funciton is not in debug mode
-    if (localdir && !message.debug && message.watch) {
-      fs.watch(localdir, {
-        recursive: true
-      }, (event, filename) => {
-        // Ignore node_modules
-        if (Array.isArray(message.watchIgnore)) {
-          for (let i = 0; i < message.watchIgnore.length; i++) {
-            if ((new RegExp(message.watchIgnore[i])).test(filename)) {
-              return;
-            }
-          }
-        }
+    if (localdir && message.watch) {
```

Comment:
Why change this conditional? Reloading the function while in debug mode will break whatever debugging the user is doing if a file changes while they're debugging.

Reply:
Thanks for reviewing! I think I wanted to test/debug the block of code, removed the conditional and then forgot to add it back (as you do :laughing:) - good catch.